installer/datafiles/Linux.data

```diff
@@ -58,9 +60,14 @@ fi
 #if DISABLE_PORT != true
     # If we're called for upgrade, don't do anything
     if ${{PERFORMING_UPGRADE_NOT}}; then
-        # Remove port 1270 from the list of ports that OMI will listen on
-        /opt/omi/bin/omiconfigeditor httpsport -r 1270 < /etc/opt/omi/conf/omiserver.conf > /etc/opt/omi/conf/omiserver.conf_temp
-        mv /etc/opt/omi/conf/omiserver.conf_temp /etc/opt/omi/conf/omiserver.conf
+        # Check if port 1270 is open
+        ${{OMI_CONF_EDITOR}} httpsport -q 1270 < ${{OMI_CONF_FILE}} > /dev/null 2>&1
+        if [ $? -eq 0 ]; then
+            touch /etc/opt/microsoft/scx/conf/scxagent-enable-port
+            # Remove port 1270 from the list of ports that OMI will listen on
```

Comment:
Why? What's the purpose of this? Why not just leave it?

Reply:
Port 1270 is used only by SCOM. So when SCX is uninstalled we can assume that port is no longer required to be open and we should close following security best practices.
Also we cannot assume that OMI will get uninstalled(hence closing the port) as part of SCX uninstall as other products (like CM) may be installed that will still require OMI.
Let me know if you see any concerns here.

Reply:
Duh. Sorry.